### PR TITLE
sys-process/glances: version bump to 2.7

### PR DESCRIPTION
**Pull request by soredake**

### CODE REVIEW

File: sys-process/glances/glances-2.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5}} )
+PYTHON_REQ_USE="ncurses"
+
+inherit distutils-r1 eutils linux-info
+
+MYPN=Glances
+MYP=${MYPN}-${PV}
```
Comment (SoapGentoo) [inline sys-process/glances/glances-2.7.ebuild]: Instead of `${MYP}`, use `${P^}` in EAPI 6, and then get rid of all the `MYPN` and `MYP`'s

**Comment by soredake** [comment]
Done

**Comment by gktrk** [comment]
Would you be interested in proxy-maintaining this package?

**Comment by soredake** [comment]
I can only do version bumps, if this is enough, i can.

**Comment by SoapGentoo** [comment]
@soredake can you add yourself to `metadata.xml`? See the format: https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide

**Comment by soredake** [comment]
Added myself as maintainer.

File: sys-process/glances/glances-2.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	# add an intended file from original data set from setup.py to DOCS
+	local DOCS=( README.rst conf/glances.conf )
+	# setup for pre-built html docs in setup.py
+	use doc && local HTML_DOCS=( docs/_build/html/. )
+
+	distutils-r1_python_install_all
+}
+
+pkg_postinst() {
+	optfeature "Action script feature"				dev-python/pystache
```
Comment (mgorny) [inline sys-process/glances/glances-2.7.ebuild]: Looks like misfired attempt at alignment. Pleae don't do that.

Comment (soredake) [inline sys-process/glances/glances-2.7.ebuild]: Fixed.

**Pull request closed without merging**